tests/integration/corrupt-dump.tcl

```diff
@@ -68,6 +69,16 @@ test {corrupt payload: valid zipped hash header, dup records} {
         r config set hash-max-ziplist-entries 1
         # cause an assertion when converting to hash table
         catch {r hset key b b}
+        verify_log_message 0 "*listpack with dup elements dump*" 0
+    }
+
+    start_server [list overrides [list loglevel verbose use-exit-on-panic yes crash-memcheck-enabled no] ] {
```

Comment:
let's keep the convention of a 1/1 test/server ratio in this file.
i.e. each test comes with it's own server, and each server with it's own test.
i did this since we're tasting assertions and crashes..

p.s. again here, with deep sanitization, we don't expect a RESTORE command to crash, we'd expect it to fail gracefully.